### ISSUE TITLE
Indent issue with 3 const and multi line declaration

### ISSUE THREAD

**Issue by MikaAK** [eslint/eslint#3673]
ESlint Version `1.2.1` 

I think this has something to do with a similar bug here #3498 

**Removing the CAT constant will fix the lint error**

### Test Code

```javascript
const BILL = 5,
      BOB  = 6,
      CAT  = 213

var billyBob = {
  name: BILL,
  friend: BOB,
  bill() {
    return CAT
  }
}

billyBob.bill()
```

### Error
```
  5:1  error  Expected indentation of 6 space characters but found 0  indent
```

**Comment by eslintbot** [comment]
Thanks for the issue! We get a lot of issues, so this message is automatically posted to each one to help you check that you've included all of the information we need to help you.

Reporting a bug? Please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
1. The source code that caused the problem
1. The configuration you're using (for the rule or your entire config file)
1. The actual ESLint output complete with line numbers

Requesting a new rule? Please be sure to include:

1. The use case for the rule - what is it trying to prevent or flag?
1. Whether the rule is trying to prevent an error or is purely stylistic
1. Why you believe this rule is generic enough to be included

Requesting a feature? Please be sure to include:

1. The problem you want to solve (don't mention the solution)
1. Your take on the correct solution to problem

Including this information in your issue helps us to triage it and get you a response as quickly as possible.

Thanks!

**Comment by nzakas** [comment]
Can you try adding a semicolon after `CAT` and see if that fixes it?

**Comment by MikaAK** [comment]
@nzakas yes indeed this fixes it. My project however doesn't use semicolons!

**Comment by nzakas** [comment]
See the pain no semicolons causes! :)

We've seen similar things before due to the parsing differences when there's a semicolon vs. when there's not, that's why I was asking.

**Comment by gyandeeps** [comment]
Duplicate of https://github.com/eslint/eslint/issues/3498
Released on `1.3.0`

**Comment by nzakas** [comment]
Thanks @gyandeeps. You can feel free to close issues that are duplicates.

**Comment by MikaAK** [comment]
@nzakas @gyandeeps I'm still having this issue with `v1.3.1`. A semicolon still fixes this problem. 
### Error
```
3:16  error  Expected indentation of 2 space characters but found 6  indent
```
### Test 
```javascript
var Command = function() {
  var fileList = [],
      files = []

  files.concat(fileList)
}


Command()
```

**Comment by gyandeeps** [comment]
What is your config for the rule?

**Comment by MikaAK** [comment]
This was my config for the rule, I could also be doing it wrong 

```
"indent": [2, 2, {"VariableDeclarator": { "var": 2, "let": 2, "const": 3}}]
```

**Comment by gyandeeps** [comment]
Yup thats an issue. 
Can you please log a new issue with all the details you just gave me?
thanks

**Comment by MikaAK** [comment]
Will do! Thanks for the confirmation

**Comment by CestDiego** [comment]
How do we get this to work with `4` spaces indentation instead of `2`? as you can see in the rule above there is a hard requirement to use 2 spaces indentation if I use that all my other code starts to have linting errors because I use 4 spaces for indentation @gyandeeps
